apps/web/src/components/FeatureFlag.tsx

```diff
@@ -59,44 +59,44 @@ const HistoryButton = ({ flagValueId }: { flagValueId: string }) => {
               <FaHistory />
             </button>
           </TooltipTrigger>
-        </Popover.Trigger>
-        <Popover.Portal>
-          <Popover.Content
-            className="max-w-lg select-none rounded-[4px] bg-gray-800 px-[15px] py-[10px] text-[15px] leading-none text-pink-50 shadow-md will-change-[transform,opacity] focus:shadow-md data-[state=open]:data-[side=bottom]:animate-slideUpAndFade data-[state=open]:data-[side=left]:animate-slideRightAndFade data-[state=open]:data-[side=right]:animate-slideLeftAndFade data-[state=open]:data-[side=top]:animate-slideDownAndFade"
-            sideOffset={5}
-          >
-            {isLoading && <LoadingSpinner />}
-            {data !== undefined && (
-              <>
-                <p className="text-xs">Edited {data.length} times</p>
-                <hr className="-mx-2 my-1 border-gray-700" />
-                <div className="space-y-4">
-                  {data.map((history) => (
-                    <div
-                      key={history.id}
-                      className="flex items-center space-x-3 "
-                    >
+        </PopoverTrigger>
+
+        <PopoverContent
+          className="w-full max-w-xl select-none text-sm"
+          sideOffset={5}
+        >
+          {isLoading && <LoadingSpinner />}
+          {data !== undefined && (
+            <>
+              <p className="text-xs">Edited {data.length} times</p>
+              <hr className="-mx-2 my-1 border-gray-700" />
+              <div className="space-y-4">
+                {data.map((history) => (
```

Comment:
Just a small detail, but when I change the value for a flag/remote config a lot of times, the history overflows and I can't read all entries anymore:
![image](https://github.com/tryabby/abby/assets/29003122/ae0c27e1-d5ce-402f-ba37-752a302981c3)
Maybe we should give this Popover a max-height and make it scrollable?

Reply:
Good catch, updated!